skydriver/__main__.py

```diff
@@ -6,6 +6,33 @@
 from .config import ENV, LOGGER, config_logging
 
 
+async def database_schema_migration(motor_client) -> None:  # type: ignore[no-untyped-def]
+    from dacite import from_dict
+    from motor.motor_asyncio import AsyncIOMotorCollection
+
+    collection = AsyncIOMotorCollection(
+        motor_client[database.utils._DB_NAME],
+        database.utils._MANIFEST_COLL_NAME,
+    )
+
+    m = 0
+    for i, doc in enumerate([d async for d in collection.find({})]):
```

Comment:
You probably want to remove `_id` via a projection.  There are some fun issues if you try to update that field.

It would also be better to put the `ewms_task` filter in the find directly.